docs/book/en/src/contribute/build_from_src/android/ndk.md

```diff
@@ -47,9 +47,9 @@ int main(int argc, char *argv[]) {
   WasmEdge_ConfigureDelete(ConfCxt);
   
   /* Create the image and TFLite import objects. */
-  WasmEdge_ImportObjectContext *ImageImpObj = WasmEdge_Image_ImportObjectCreate();
-  WasmEdge_ImportObjectContext *TFLiteImpObj = WasmEdge_TensorflowLite_ImportObjectCreate();
-  WasmEdge_ImportObjectContext *TFDummyImpObj = WasmEdge_Tensorflow_ImportObjectCreateDummy();
+  WasmEdge_ModuleInstanceContext *ImageImpObj = WasmEdge_Image_ImportObjectCreate();
```

Comment:
Should be `WasmEdge_Image_ModuleInstanceCreate()`.

Reply:
oops ... forgot to copy that to my repo ... will do